controller/model/posture_check_model_windows_domain.go

```diff
@@ -26,6 +26,20 @@ type PostureCheckWindowsDomains struct {
 	Domains []string
 }
 
+func (p *PostureCheckWindowsDomains) Evaluate(pd *PostureData) bool {
+	if pd.Domain.TimedOut {
+		return false
```

Comment:
on all these TimedOut checks, is it worthwhile to log/know when those types of timeouts occur? not sure if it is exceptional, expected, or if it could be a flood of logs so i can understand if it's not logged on purpose but seems like that sort of information will be hard to debug (maybe)

Reply:
Timeouts are expected to happen if the client stops sending data. The frequency could be large or very little.

If you want to know if you have timed out data you can review the posture data on any identity and it will clearly show the data timed out.